projects/demo-playwright/tests/kit/input-number/input-number.spec.ts

```diff
@@ -352,25 +361,25 @@ test.describe('InputNumber', () => {
             await input.fill('42,00');
             await expect(input).toHaveJSProperty('selectionStart', 5);
             await expect(input).toHaveJSProperty('selectionEnd', 5);
-            await expect(example).toHaveScreenshot('28-input-number.png');
+            await expect(example).toHaveScreenshot('29-input-number.png');
         });
 
-        test('Value 42 (decimal=never) => 42', async ({page}) => {
-            await tuiGoto(page, '/components/input-number/API?precision=2&decimal=never');
+        test('Value 42,1 (precision=0) => 42', async ({page}) => {
+            await tuiGoto(page, '/components/input-number/API?precision=0');
 
-            await input.fill('42');
-            await expect(example).toHaveScreenshot('29-input-number.png');
+            await input.fill('42,1');
+            await expect(example).toHaveScreenshot('30-input-number.png');
```

Comment:
```suggestion
            await expect(input).toHaveValue('42');
            await expect(example).toHaveScreenshot('30-input-number.png');
```

(additional security level 😄 )

Reply:
Could you add it to other tests, please ?